libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -161,7 +161,7 @@ class JSONNode {
 
 
     json_string as_string(void) const json_nothrow json_read_priority;
-    int as_int(void) const json_nothrow json_read_priority;
+    json_int_t as_int(void) const json_nothrow json_read_priority;
```

Comment:
If we expand it I would propose to use `int64_t` from `cstdint` because it is more explicit than using `long long long something something something`.